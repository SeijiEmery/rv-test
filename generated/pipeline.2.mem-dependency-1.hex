0000000    23  b0  84  00  03  b9  04  00  b3  09  24  01  13  00  00  00
0000020    13  00  00  00  13  00  00  00  13  00  00  00  13  00  00  00
0000040    73  00  10  00                                                
0000044
