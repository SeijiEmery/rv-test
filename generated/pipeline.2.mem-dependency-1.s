sd s0, 0(s1)
	ld s2, 0(s1)
	add s3, s0, s2
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
