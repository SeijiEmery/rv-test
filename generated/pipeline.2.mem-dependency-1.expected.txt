tests/pipeline.test.s:22
R8 = 1000
R9 = 2048
R18 = 1000
R19 = 2000
